msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.MacDev.Tasks {
+	public abstract class CreateBindingResourcePackageBase : Task {
+		public string SessionId { get; set; }
+		
+		[Required]
+		public string OutputPath { get; set; }
```

Comment:
is not `ITaskItem` preferred to `string` for paths ?

Reply:
I think $(OutputPath) is a string, since it doesn't have any extra metadata, but the conversion would happen once so I think it's fine. @jstedfast thoughts?

Reply:
Since this is an input property it can be string, ITaskItem is needed from VS for all [Output] props. In the case of CompileITunesMetadataTaskBase we're using the OutpuPath property also as output.

Reply:
FWIW, a lot of these are `string` because when I wrote them, I had no idea what I was doing and it was good enough for Mac-only builds. Once VS Windows started using the targets, we've slowly migrated some of them to `ITaskItem` as needed.

If @emaf says `string` is ok in this case, then I'm fine with that.

At the end of the day, what VS Windows guys need is what we do :-)